src/audio/buffer.c

```diff
@@ -312,8 +312,14 @@ void buffer_attach(struct comp_buffer *buffer, struct list_item *head, int dir)
 	list_item_prepend(list, head);
 	if (further_buffers_exist)
 		dcache_invalidate_region(needs_sync, sizeof(struct list_item));
-	/* no dirty cache lines exist for this buffer yet, no need to write back */
+#if CONFIG_INTEL
+	/* no dirty cache lines exist for this buffer yet, no need to write back
```

Comment:
I think this comment line can be dropped now, you probably forgot to remove it

Reply:
@lyakh I just copied the code snippet you pasted. (https://github.com/thesofproject/sof/pull/7925#discussion_r1263343320)
 
Do you mean I should drop the line "no dirty cache lines exist for this buffer yet, no need to write back"?

Reply:
@wenliangwu yes, the diff I posted had a `-` in front of that line, that meant that that line had to be dropped